feat: re-structured language command with sub commands

alvarosabu opened the pull request:
- Changed command name from `pull-languages` to `languages` across the codebase for consistency and add sub command 'pull'
- Updated resolved path to `.storyblok/languages/spaceId/filename.suffix.json` by default
- Updated launch configuration to reflect the new command name.
- Introduced new actions and constants for managing languages, including fetching and saving language data.
- Added tests for the new languages command and its actions.
- Removed obsolete `pull-languages` command and its associated tests.

## Previously:

```
storyblok pull-languages -s <spaceId>
```

Saves to `.storyblok/languages/languages.spaceId.json`


## Current

```
storyblok languages pull -s <spaceId>
```

Saves to `.storyblok/languages/spaceId/languages.json`

alvarosabu committed:
feat: re-structured language command with sub commands

- Changed command name from `pull-languages` to `languages` across the codebase for consistency and add sub command 'pull'
- Updated resolved path to `.storyblok/languages/spaceId/filename.suffix.json` by default
- Updated launch configuration to reflect the new command name.
- Introduced new actions and constants for managing languages, including fetching and saving language data.
- Added tests for the new languages command and its actions.
- Removed obsolete `pull-languages` command and its associated tests.

edodusi commented on src/commands/languages/index.ts:
```diff
@@ -2,25 +2,30 @@ import { colorPalette, commands } from '../../constants'
-  .command('pull-languages')
-  .description(`Download your space's languages schema as json`)
+export const languagesCommand = program
+  .command('languages')
```
@alvarosabu can I suggest to add an alias() `lang` here? I'm a fan of shortcuts 😄

alvarosabu replied:
Done @edodusi 😉, you can't say I don't spoil you haha

edodusi replied:
niiiiice 🫶

edodusi reviewed (approved):
Just a quick suggestion, for the rest LGTM!

alvarosabu committed:
feat: add alias for languages command

- Introduced an alias 'lang' for the 'languages' command to enhance usability and streamline command input.
